runtime/service/src/intTest/java/org/apache/polaris/service/it/RestCatalogMinIOSpecialIT.java

```diff
@@ -86,7 +85,6 @@
 @QuarkusIntegrationTest
 @TestProfile(RestCatalogMinIOSpecialIT.Profile.class)
 @ExtendWith(MinioExtension.class)
-@ExtendWith(PolarisIntegrationTestExtension.class)
```

Comment:
This extension should not be enabled in IT tests annotated with `@QuarkusIntegrationTest`.